src/commands/BootstrapCommand.js

```diff
@@ -27,7 +40,7 @@ export default class BootstrapCommand extends Command {
     this.progressBar.init(this.packages.length);
     this.logger.info("Linking all dependencies");
 
-    async.parallelLimit(this.packages.map(pkg => done => {
+    async.parallelLimit(this.packages.filter(pkg => !pkg.ignored).map(pkg => done => {
```

Comment:
In other words, don't add an ignored package to the packages array in `PackageUtilities.getPackages`?